lldb/test/API/functionalities/breakpoint/breakpoint_conditions/TestBreakpointConditions.py

```diff
@@ -42,17 +52,21 @@ def setUp(self):
             "main.c", "// Find the line number of c's parent call here."
         )
 
-    def breakpoint_conditions(self, inline=False):
+    def breakpoint_conditions(self, inline=False, language=None):
         """Exercise breakpoint condition with 'breakpoint modify -c <expr> id'."""
         exe = self.getBuildArtifact("a.out")
         self.runCmd("file " + exe, CURRENT_EXECUTABLE_SET)
 
+        cmd_args = "-c 'val == 3'"
```

Comment:
This seems like it would still pass if the `-Y` option were to be ignored, since it's a valid conditional in Swift and C.

Reply:
You're right, I changed the condition to use `nullptr` so that it fails to parse if the language is wrong.